language-server/package.json

```diff
@@ -1,37 +1,5 @@
 {
-  "name": "vscode-json-languageserver",
-  "description": "JSON language server",
-  "version": "1.3.1",
-  "author": "Microsoft Corporation",
-  "license": "MIT",
-  "engines": {
-    "node": "*"
-  },
-  "bin": {
-    "vscode-json-languageserver": "./bin/vscode-json-languageserver"
-  },
-  "main": "./out/node/jsonServerMain",
   "dependencies": {
```

Comment:
Wouldn't be too bad to keep some of the fields that are "required" (or trigger warning when missing).